fendermint/vm/interpreter/src/fvm/gas/actor.rs

```diff
@@ -45,9 +46,9 @@ impl ActorGasMarket {
             to: GAS_MARKET_ACTOR_ADDR,
             sequence: block_height as u64,
             // exclude this from gas restriction
-            gas_limit: u64::MAX,
-            method_num: fendermint_actor_gas_market::Method::CurrentGasReading as u64,
-            params: fvm_ipld_encoding::RawBytes::serialize(())?,
+            gas_limit: i64::MAX as u64,
```

Comment:
What's the deal here?

Reply:
You mean why `i64::MAX`? Because `u64::MAX` will cause an error, I think internally it's checking the u64 can be casted safely to u64.